arquillian/RESTEASY-1630-jetty-resteasy-servlet-initializer/src/test/java/org/jboss/resteasy/test/resteasy1630/ServletInitializerTwoTest.java

```diff
@@ -0,0 +1,54 @@
+package org.jboss.resteasy.test.resteasy1630;
+
+import java.net.URI;
+import javax.ws.rs.core.Response;
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.container.test.api.RunAsClient;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.arquillian.test.api.ArquillianResource;
+import org.jboss.resteasy.client.jaxrs.ResteasyClientBuilder;
+import org.jboss.resteasy.resteasy1630.TestApplicationPath;
+import org.jboss.resteasy.resteasy1630.TestResource;
+import org.jboss.shrinkwrap.api.Archive;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static org.junit.Assert.assertEquals;
+
+/**
```

Comment:
Please add some javadoc here to explain the test. Basically, the reason why we are adding this test is to show how the deployment based on the resteasy-servlet-initializer approach works. So, those looking at this test need to either find the resteasy-servlet-initializer explicitly added to the deployment created using ShrinkWrap, or read a note explaining that Arquillian is automatically adding the resteasy-servlet-initializer dependency to the deployment as a consequence of that dependency having been specified in the pom.xml for this test project. Without that, most readers will simply miss the point. Thanks.